catkin_home/src/devices/src/InputAudio.cpp

```diff
@@ -9,6 +9,7 @@ void onAudioCallback(const audio_common_msgs::AudioData::ConstPtr msg){
     cout<<"I heard "<<endl;
     vector<uint8_t> test;
     test.push_back(1);
+   //Define otro mensaje que va a enviar
```

Comment:
Please use the next time the word "TODO:" the next time if this is a to-do. Also, I think the call of the parser will be in action_selectors/hear.py instead.

Reply:
okok this file is outdated though so the new version found in the develop branch will be kept instead.